test/LocalDataSource.spec.tsx

```diff
@@ -309,6 +310,19 @@ describe('LocalDataSource', () => {
         done();
       });
     });
+
+    // Not Ends With
+    it('should return a payload with records where CustomerName not ends with \'a\'', (done) => {
+      dataSource.columns[1].Filter.Text = 'a';
+      dataSource.columns[1].Filter.Operator = 'NotEndsWith';
```

Comment:
There is not an enum for Filter Operator?

Reply:
Updated, @geoperez